src/providers/bettersqlite.js

```diff
@@ -0,0 +1,239 @@
+const SettingProvider = require('./base');
+
+/**
+ * Uses an SQLite database to store settings with guilds
+ * @extends {SettingProvider}
+ */
+class BetterSQLiteProvider extends SettingProvider {
+	/**
+	 * @external BetterSQLiteDatabase
+	 * @see {@link https://www.npmjs.com/package/better-sqlite3}
+	 */
+
+	/**
+	 * @param {BetterSQLiteDatabase} conn - Database Connection for the provider
+	 */
+	constructor(conn) {
+		super();
+
+		/**
+		 * Database that will be used for storing/retrieving settings
+		 * @type {BetterSQLiteDatabase}
+		 */
+		this.conn = conn;
+
+		/**
+		 * Client that the provider is for (set once the client is ready, after using {@link CommandoClient#setProvider})
+		 * @name BetterSQLiteProvider#client
+		 * @type {CommandoClient}
+		 * @readonly
+		 */
+		Object.defineProperty(this, 'client', { value: null, writable: true });
```

Comment:
This should come before any other properties.

Reply:
You mean move the entire block right under the `super();`? Honestly I only changed the `@name` tag there but other than that it was in this order for `sqlite.js` as well. 

Should I move it to just under `super();` for the both of them then?

Reply:
I have it locally stashed and would now look like this for both files (barring the `@name` property). Before I push this, is this correct?

```js
constructor(conn) {
    super();
 
    /**
     * Client that the provider is for (set once the client is ready, after using {@link CommandoClient#setProvider})
     * @name SyncSQLiteProvider#client
     * @type {CommandoClient}
     * @readonly
     */
    Object.defineProperty(this, 'client', { value: null, writable: true });
 
    /**
     * Database that will be used for storing/retrieving settings
     * @type {SyncSQLiteDatabase}
     */
    this.conn = conn;
 
    /**
     * Settings cached in memory, mapped by guild ID (or 'global')
     * @type {Map}
     * @private
     */
    this.settings = new Map();
```